### PR TITLE
Merging the initial CC code to the one developed at NERSC

### PR DESCRIPTION
**Pull request by johannct**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,33 @@ Install the requisite packages into the virtual environment
...
+FINK_SERVER
+FINK_TOPIC
+```
 
 ## Local Database Server
 
-Getting a dockerized  database up and running is a pre-requisite to all of the local development methods. Here's how:
+Getting a dockerized  database up and running is a required. Here's how:
 ```bash
-export DB_HOST=127.0.0.1
```
Comment (AlexGKim) [inline README.md:25]: I need this line personally.

Comment (AlexGKim) [inline README.md:25]: I need this line

**Review by AlexGKim — COMMENTED** [review]
Can you keep the DB_HOST in the README?

File: README.md (hunk 1 above)
Comment (johannct) [reply README.md:25]: hmm, this was not intentional, no idea how I ended up with a different README.md than main.

**Pull request merged**